kythe/go/platform/vfs/vfs.go

```diff
@@ -41,6 +41,8 @@ type Interface interface {
 	Writer
 }
 
+// NamedWriteCloser composes io.WriteCloser and access to its "name". For
```

Comment:
What do you think about calling this more simply `TempFile`?

```go
func (localFS) TempFile(ctx context.Context, dir, pattern string) (TempFile, error) { …
```

Not that this is inaccurate, but it's lower level when what we care about is that it's the contact surface for a temporary file. YMMV.

Reply:
I like it, except it conflicts with your preferred function name. You OK with "CreateTempFile" ?

Reply:
It doesn't conflict—the function name is inside the type.

Reply:
Oh, I see, there is a top-level helper function.

Yeah, I mean that works too.